apps/test/unit/lib/kits/maker/dropletConfigTest.js

```diff
@@ -74,7 +74,8 @@ describe('getBoardEventDropdownForParam', () => {
     ]);
   });
 
-  describe('touchPads', () => {
+  // TODO (captouch): Re-enable
+  describe.skip('touchPads', () => {
```

Comment:
Oh, you haven't seen this?  Yeah, nice mocha features we don't use much:

* [Pending test cases](https://mochajs.org/#pending-tests)
* [`skip()`](https://mochajs.org/#inclusive-tests) to skip tests
* [`only()`](https://mochajs.org/#exclusive-tests) to run only one test (great for iterating)